eventually/utils/validators.py

```diff
@@ -312,3 +316,23 @@ def comment_data_validator(data):
         return
 
     return True
+
+
+def profile_data_validator(data):
+    """
+    Function that validation incoming request.body
+    :param request_body: data that need to validate.
+    :type data: HttpRequest
+    :return: data if data is valid and `None` if it is not.
+    """
+
+    if not string_validator(data.get("hobby")):
```

Comment:
what happens if one of that field value will be `None`?

Reply:
chanched